resource-aggregate/events/resourceChanged_test.go

```diff
@@ -30,7 +30,8 @@ var testEventResourceChanged events.ResourceChanged = events.ResourceChanged{
 		ConnectionId: "con1",
 		Sequence:     1,
 	},
-	Status: commands.Status_ACCEPTED,
+	Status:        commands.Status_ACCEPTED,
+	ResourceTypes: []string{"type1", "type2"},
```

Comment:
I'd extend the tests for `CopyData` like this for other event types as well (`ResourceCreatePending`, `ResourceCreated`, ...)

Reply:
added